packages/rancher-logging/generated-changes/patch/templates/_helpers.tpl.patch

```diff
@@ -22,3 +22,11 @@
 +  {{- end }}
 +  {{- end }}
 +{{- end }}
++
++{{- define "windowsPathPrefix" }}
++{{- $pathPrefix := (required "Must provide value for .Values.global.cattle.rkeWindowsPathPrefix" .Values.global.cattle.rkeWindowsPathPrefix) -}}
```

Comment:
`.Values.global.cattle.rkeWindowsPathPrefix` can be empty string and should be defaulted. The dashboard UI will specifically override the default for `""` due to the way it is currently defined.

Reply:
This is an old commit. I'll squash.